hera_pspec/container.py

```diff
@@ -3,21 +3,32 @@
 from hera_pspec import uvpspec, version, utils
 import argparse
 import time
+from functools import wraps
 
 
 def transactional(fn):
     """
     Handle 'transactional' operations on PSpecContainer, where the HDF5 file is 
     opened and then closed again for every operation. This is done when 
     keep_open = False.
+
+    For calling a @transactional function within another @transactional function,
+    feed the kwarg nested=True in the nested function call, and it will not close
+    the container upon exit even if keep_open = False, until the outer-most
+    @transactional function exits, in which case it will close the container
+    if keep_open = False.
     """
+    @wraps(fn)
```

Comment:
a neat thing I read on stackoverflow: In Python 3 this allows the function signature and doc-string to passed along with the wrapped function, and in Python 2 only the docstring is passed along (not the signature). This means that for example `help(PSpecContainer.set_pspec)` shows the the actual `set_pspec` metadata, rather than the `wrapper()` metadata as it would without the `@wraps` call.